CHANGELOG.md

```diff
@@ -18,4 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
-## [1.0.0] - 2021.12.12
\ No newline at end of file
+## [1.0.0] - 2021.12.12
+
+## [1.1] - TBD
```

Comment:
Its general protocol to put the changes in the unreleased section at the top until a release is performed, at which point you would give it a version number and release date, and copy the categories up to the top and label them under `[Unreleased]`.

Reply:
Ah, fair. Changelog's are kinda new to me. The only changelogs we really do at work are when we do code review.

Reply:
Fixed!

Reply:
The change you made is better, but you forgot to put it under the appropriate category. I imagine this would go under the 'added' category header, under the unreleased.